frontend/src/scenes/plugins/Plugins.scss

```diff
@@ -2,10 +2,6 @@
     cursor: default;
 }
 
-.Plugins__Popconfirm {
-    z-index: var(--z-plugins-popconfirm);
```

Comment:
So we no longer need the `--z-plugins-popconfirm` var – let's clean up by removing it from `vars.scss`